<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cnn</class>
 <widget class="QWidget" name="cnn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>1014</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 11pt &quot;Seagoe UI&quot;;
color:#606060;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>60</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>60</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font:14pt &quot;Seagoe UI&quot;;
font:bold;</string>
     </property>
     <property name="text">
      <string>Convolutional Neural Networks</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>25</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Select option:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="trainRadioButton">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Train</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="testRadioButton">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Test</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="trainProgressBar">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="rightMargin">
          <number>55</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="showResultsButton">
           <property name="minimumSize">
            <size>
             <width>445</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>420</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 12pt;
</string>
           </property>
           <property name="text">
            <string>Show Results</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="valueLabel">
       <property name="minimumSize">
        <size>
         <width>1000</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt;
</string>
       </property>
       <property name="text">
        <string>text</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>700</height>
      </size>
     </property>
     <widget class="QWidget" name="plotPage">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>600</height>
       </size>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>0</y>
         <width>1181</width>
         <height>591</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="plotGridLayout"/>
      </widget>
     </widget>
     <widget class="QWidget" name="resultsPage">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1300</width>
        <height>700</height>
       </size>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>0</y>
         <width>1181</width>
         <height>602</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="accuracyButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string>Accuracy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="precRecallButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Precision and Recall</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="confusionMatrixButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string>Confusion Matrix</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="RocCurveButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string>ROC Curve</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QStackedWidget" name="resultsWidget">
          <property name="minimumSize">
           <size>
            <width>650</width>
            <height>600</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1030</width>
            <height>700</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="accPage">
           <widget class="QWidget" name="horizontalLayoutWidget">
            <property name="geometry">
             <rect>
              <x>-1</x>
              <y>-1</y>
              <width>1031</width>
              <height>601</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QWidget" name="widget_2" native="true">
               <property name="maximumSize">
                <size>
                 <width>520</width>
                 <height>600</height>
                </size>
               </property>
               <widget class="QWidget" name="gridLayoutWidget_2">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>511</width>
                  <height>601</height>
                 </rect>
                </property>
                <layout class="QGridLayout" name="trainGridLayout"/>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget" native="true">
               <property name="maximumSize">
                <size>
                 <width>520</width>
                 <height>600</height>
                </size>
               </property>
               <widget class="QWidget" name="gridLayoutWidget_5">
                <property name="geometry">
                 <rect>
                  <x>-1</x>
                  <y>-1</y>
                  <width>511</width>
                  <height>601</height>
                 </rect>
                </property>
                <layout class="QGridLayout" name="testGridLayout"/>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="precRecallPage">
           <widget class="QWidget" name="gridLayoutWidget_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>1021</width>
              <height>601</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="precRecallGridLayout"/>
           </widget>
          </widget>
          <widget class="QWidget" name="confusionMatrixPage">
           <widget class="QWidget" name="gridLayoutWidget_4">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>1021</width>
              <height>601</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="confusionMatrixLabel">
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="rocCurvePage">
           <widget class="QWidget" name="gridLayoutWidget_6">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>-1</y>
              <width>1021</width>
              <height>601</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="rocCurveLabel">
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="waitPage">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="waitLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt;
</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="compareButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt;
</string>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
